Sources/TootSDK/Models/Translation.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Translation.swift
+//
+//
+//  Created by Philip Chu on 1/30/24.
+//
+
+import Foundation
+
+public struct Translation: Codable {
```

Comment:
I think none of the translation attributes are guaranteed to be provided. The fields should be optional.